assets/queries/ansible/aws/cloudformation_stack_notifications/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "cloudformation_stack_notifications",
+  "queryName": "AWS CloudFormation Stack Notifications",
```

Comment:
Replace the name with "AWS CloudFormation Stack Notifications Disabled", to be easier to understand what the vulnerability is.